molecule/agent-uninstall-test/molecule.yml

```diff
@@ -0,0 +1,88 @@
+---
+# This file is the central configuration entrypoint for Molecule. With this file, you can configure each tool that Molecule will employ when testing the role.
+dependency:
+  name: galaxy
+driver:
+  name: delegated
+  gcp_service_account_key: ${GOOGLE_APPLICATION_CREDENTIALS}
+  gcp_project_id: ${PROJECT}
+  zone: ${ZONE}
+  region: ${REGION}
+  ssh_user: ${SSH_USER}
+  ssh_pub_key_file: "${SSH_KEY_FILE}.pub"
+  ssh_key_file: ${SSH_KEY_FILE}
+  network_name: "ansible-network-${AGENT_TYPE}-${MOLECULE_SCENARIO_NAME}"
+  firewall_name: ansible-firewall
```

Comment:
network_name and firewall_name are unused now, right?

Reply:
Yep, fixed.